### ISSUE TITLE
Fix font styling in PDF

### ISSUE THREAD

**Issue by barelyhuman** [barelyhuman/mark#16]
Currently the PDF doesn't match the font style or render style from the page, 
need to add a service for that or fix it in the `html2pdf.js` package

**Comment by itsmohmans** [comment]
I'm not sure if this is the same issue you're referring to, but the text in the generated PDF is just an image of the actual text, which means it can't be selected, and it's pixelated on zoom.

You can see the difference and how it should be if you convert the page to pdf using browser's print functionality (`ctrl + p`).

Edit: it seems to be an issue in `html2pdf.js`

Edit 2: Possible solution:  Use [`node-wkhtmltopdf`](node-wkhtmltopdf) in a server-side environment, it should work with Nuxt server-side to generate the pdf.

**Comment by barelyhuman** [comment]
Yep, there’s a local branch with a reconstructed pdf over an edge function but I’m planning to add in auth and tabs to it soon. Will probably fix it with that
